### PR TITLE
DOM Solution - Rajashree

### PR DESCRIPTION
**Pull request by rajashreemaiya**
Completed the basic challenge.

Yet to complete the extra credit part.

@jaybobo @sampriddy @sjreich

### CODE REVIEW

File: top-nav-drill/index.html [hunk 1]
```diff
@@ -27,9 +27,10 @@
 
     <section>
     <h1>A Page Title</h1>
+    <!-- <hr> -->
     <section>
```
Comment (paradime) [inline top-nav-drill/index.html]: This is pretty hard to figure out if this section is within section

File: top-nav-drill/main.css [hunk 2]
```diff
@@ -2,4 +2,80 @@
...
+  background-color: black;
+  height: 45px;
+}
+
+ul
+{
+  margin-top: 7px;
+}
+nav
+{
```
Comment (paradime) [inline top-nav-drill/main.css]: Some of your formatting here is a little hard to follow. Think the most common implementation is

<CSS Selector>{
  <field>: value;
}

etc.

**Comment by paradime** [comment]
Nice job! I would recommend trying to implement this in HAML and SASS next

**Comment by rajashreemaiya** [comment]
Completed Haml and Sass.

@paradime @jaybobo @sjreich @sampriddy
